core/capabilities/webapi/target/types.go

```diff
@@ -0,0 +1,19 @@
+package target
+
+const (
+	AllAtOnce  string = "AllAtOnce"
+	RoundRobin string = "RoundRobin"
+)
+
+type WorkflowInput struct {
+	URL     string            `json:"url"`               // URL to query, only http and https protocols are supported.
+	Method  string            `json:"method,omitempty"`  // HTTP verb, defaults to GET.
+	Headers map[string]string `json:"headers,omitempty"` // HTTP headers, defaults to empty.
+	Body    string            `json:"body,omitempty"`    // Base64-encoded binary body, defaults to empty.
+}
+
+type WorkflowConfig struct {
```

Comment:
Only "Config", drop the "Workflow".

Reply:
Isn't `WorkflowConfig` a good name for this? Thinking `Config` should be the config for the capability and the handler and `WorkflowConfig` or`UserWorkflowConfig` is the user-defined workflow config?